stdlib/3/lib2to3/pgen2/parse.pyi

```diff
@@ -0,0 +1,30 @@
+# Stubs for lib2to3.pgen2.parse (Python 3.6)
+
+from typing import Any, List, Optional, Sequence, Set, Tuple
+
+from lib2to3.pgen2.driver import _Convert, _RawNode
+from lib2to3.pgen2.grammar import Grammar, _States
+from lib2to3.pytree import _NL
+
+_Context = Sequence[Any]
+
+class ParseError(Exception):
+    msg: str
+    type: int
+    value: Optional[str]
+    context: _Context
+    def __init__(self, msg: str, type: int, value: Optional[str], context: _Context) -> None: ...
+
+class Parser:
+    grammar: Grammar
+    convert: _Convert
+    stack: List[Tuple[_States, int, _RawNode]]
```

Comment:
The first element of the tuple should be `Tuple[_DFA, Dict[int, int]]`, same as the value type of Grammar.dfas.